src/components/Card/Card.tsx

```diff
@@ -67,34 +79,8 @@ const spacingStyles = ({ theme, spacing }) => {
   `;
 };
 
-/**
- * Card component that is used for displaying content on a grid.
- */
-const Card = styled('div', {
-  shouldForwardProp: (prop) => isPropValid(prop) && prop !== 'spacing',
-})(baseStyles, shadowStyles, spacingStyles);
-
-Card.propTypes = {
-  /**
-   * The shadow depth of the Card.
-   */
-  shadow: PropTypes.oneOf(['single', 'double', 'triple']),
-  /**
-   * The padding of the Card.
-   */
-  spacing: PropTypes.oneOf(['mega', 'giga']),
-  /**
-   * Content to be rendered inside the Card.
-   */
-  children: childrenPropType,
-};
-
-Card.defaultProps = {
-  spacing: 'giga',
-  shadow: 'single',
-};
-
-/**
- * @component
- */
-export default Card;
+export const Card: FC<CardProps> = styled('div')<CardProps>(
+  baseStyles,
+  shadowStyles,
+  spacingStyles,
+);
```

Comment:
Is there a reason why you removed the `shouldForwardProp` option? It still seems necessary to prevent invalid HTML attributes.

Reply:
Well I thought that using typescript the prop was not necessary anymore... but thinking better, one thing doesn't relate with the other 🤣